README.md

```diff
@@ -1,53 +1,804 @@
-#SF-WDI-23 Homework submission
 
-Homework will be turned in to your fork of this repository daily.
 
-##Setup
 
-To set up your initial homework repository do the following:
```

Comment:
I accidentally deleted the README file the wdi file because i thought it was duplicating the one in my smaupin folder. Then I realized I had deleted an actual file so tried to download it from git and replace it before doing a commit. I think this may be an odd thing that i wasnt supposed to do, and might produce an odd result like you saw. thanks for fixing it. does that mean i need to fix it on my computer too? Not sure how to do that.